.buildkite/jobs/pipeline.android_demo_app_rn_76.yml

```diff
@@ -1,4 +1,4 @@
-  - label: ":android::react: RN .76 + Android: Demo app"
+  - label: ":android::react: (New Arch) - RN .76 + Android: Demo app"
```

Comment:
Why not testing both new arch and old arch on .76?
Is that a breaking change?

Reply:
I think it's ok for the demo app but the question is valid for the main test app; By now we know the backwards compatibility is feasible with a reasonable effort so let's take it